Tevling/Pages/Statistics.razor.cs

```diff
@@ -15,39 +49,45 @@ public partial class Statistics : ComponentBase, IAsyncDisposable
 
     private int NumberOfMonthsToReview { get; set; } = 3;
     private ChallengeMeasurement Measurement { get; set; } = ChallengeMeasurement.Distance;
-    private Dictionary<string, float[]> Distances { get; set; } = [];
-    private Dictionary<string, float[]> Elevations { get; set; } = [];
-    private Dictionary<string, float[]> Durations { get; set; } = [];
+    private List<Stats> Distances { get; set; } = [];
+    private List<Stats> Elevations { get; set; } = [];
+    private List<Stats> Durations { get; set; } = [];
```

Comment:
Unless you need to have mutable lists, I'd prefer to use `IReadOnlyList`.
```suggestion
    private IReadOnlyList<Stats> Distances { get; set; } = [];
    private IReadOnlyList<Stats> Elevations { get; set; } = [];
    private IReadOnlyList<Stats> Durations { get; set; } = [];
```

Reply:
👍 

Fixed: https://github.com/veloek/tevling/pull/55/commits/91e084790adbc143584a80201fc50eb9803bdae9